sass/layout/_sidebar-left.scss

```
@@ -924,7 +924,13 @@
         }
 
         .badge {
-            display: none;
+            // Hide the badge by giving it 0 width instead of using `display: none` since the desktop app only counts
+            // badges with a non-zero offsetHeight when counting notifications
+            margin: 0px;
+            min-width: 0px;
+            overflow: hidden;
+            padding: 0px;
+            width: 0px;
```

Nice! Interesting implementation on the Desktop app, I wonder why it is looking to ensure the badge is visible?

You can also achieve this by using:
```suggestion
            // Hide the badge by hiding it instead of using `display: none` since the desktop app only counts
            // badges with a non-zero offsetHeight when counting notifications
            position: absolute;
            visibility: hidden;
```

---

I did a bit more digging, and it's because we had two sets of badges in the app, one in the regular sidebar and another in a mobile sidebar, so we had to make sure only to count visible ones. I'm pretty sure the LHS isn't duplicated between the two any more, so that shouldn't be necessary, but that code's been in place [for 4 and a half years](https://github.com/mattermost/desktop/commit/929e5e6f88f513badddd07ef454be30cd6e1bcbb) at this point.

As for the CSS, I definitely like the simpler CSS there.